packages/cli/src/config/settings.test.ts

```diff
@@ -409,7 +484,7 @@ describe('Settings Loading and Merging', () => {
       delete process.env.WORKSPACE_ENDPOINT;
     });
 
-    it('should prioritize workspace env variables over user env variables if keys clash after resolution', () => {
```

Comment:
keep a test verifying that workspace overrides user as well as adding a test that system overrides workspace.